contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -1155,29 +1100,13 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
         }
     }
 
+    function setDemandCurve(ICurvedInterestRate _rateHelper) public onlyGuardian {
```

Comment:
should we move this to the protocol contract? This would make it a lot easier to change the curve as I do not foresee the need for a pool-specific curve due to how our interest rate curve is set up.

Reply:
interesting idea hm

Reply:
we discussed that. the reason we didn't move is because utilization rate is easier to calc on pool side.